css/style.css

```diff
@@ -9,400 +14,616 @@ body {
-    top: 12px;
-    right: 12px;
+  position: absolute;
+  top: 12px;
+  right: 12px;
 }
 .img {
-    position: relative;
+  position: absolute;
+  top: 3%;
+  right: 0;
 }
+
 .bag__title-bold {
-    font-style: normal;
-    font-weight: bold;
-    font-size: 40px;
-    line-height: 41px;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 41px;
 }
 
 .bag__title {
-    font-style: normal;
-    font-weight: bold;
-    font-size: 16px;
-    line-height: 16px;
-    letter-spacing: .07em;
-    opacity: .5;
-}
-
-.bag-text{
-    font-style: normal;
-    font-weight: normal;
-    font-size: 15px;
-    line-height: 21px;
-    opacity: .5;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 16px;
+  letter-spacing: .07em;
+  opacity: .5;
+  position: absolute;
+  width: 220px;
+  height: 17px;
+  left: 55px;
+  top: 98px;
+}
+
+.bag-text {
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 21px;
+  opacity: .5;
+  position: absolute;
+  width: 368px;
+  height: 148px;
+  left: 55px;
+  top: 179px;
+  color: #000;
 }
 
 .product {
-    font-style: normal;
-    font-weight: 500;
-    font-size: 13px;
-    line-height: 16px; 
+  font-style: normal;
+  font-weight: 500;
+  font-size: 13px;
+  line-height: 16px;
 }
 
 .bag-title-bold {
-    font-style: normal;
-    font-weight: bold;
-    font-size: 40px;
-    line-height: 41px;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 41px;
 }
 
 .bag-title {
-    font-style: normal;
-    font-weight: bold;
-    font-size: 16px;
-    line-height: 16px;
-    letter-spacing: .07em;
-    opacity: .5;  
+  font-style: normal;
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 16px;
+  letter-spacing: 1px;
+  opacity: .5;
 }
 
 .total-look__text {
-    font-style: normal;
-    font-weight: normal;
-    font-size: 15px;
-    line-height: 21px;
-    opacity: .5;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 21px;
+  opacity: .5;
 }
 
 .name {
-    font-style: normal;
-    font-weight: bold;
-    font-size: 12px;
-    line-height: 15px;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 12px;
+  line-height: 15px;
 }
 
-.new-collection__background {
-    background-size: 100% auto;
+/* Cтили 2й секции ниже */
+
+.bag {
+  background: #fae8e4;
+  height: 900px;
+  margin-top: 0;
+  width: 1440px;
 }
 
+.bag__wight {
+  position: absolute;
+  width: 398px;
+  height: 415px;
+  margin-left: 137px;
+  top: 1117px;
+}
 
-/* Медиа запросы */
+.bag__pink {
+  position: absolute;
+  width: 582px;
+  height: 844px;
+  margin-left: 231px;
+  top: 928px;
+}
 
-@media (max-width: 1270px) { 
-   .title {
-       top: 30%;
-       width: 20%;
-       font-size: 50px;
-       line-height: 52px;
-       left: 60%;
-   }
-    } 
+.card {
+  position: absolute;
+  width: 488px;
+  height: 556px;
+  margin-left: 879px;
+  top: 1068px;
+  background: #FFFFFF;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "#FFFFFF" to be "#ffffff" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/Love-metal_LOCO/pullRequest?prid=5478856)